features/bootstrap/DatasetViewContext.php

```diff
@@ -416,7 +416,21 @@ public function iHaveSetPagesizeToOn($arg1, $arg2)
         }
     }
 
+    /**
+     * @Then there is a meta tag :arg1 with value :arg2
+     */
+    public function thereIsAMetaTagWithValue($arg1, $arg2)
+    {
+        $metaNode = $this->minkContext->getSession()->getPage()->find('xpath', "//meta[@name='$arg1' and @content='$arg2']");
+        PHPUnit_Framework_Assert::assertNotNull($metaNode);
+    }
 
-
-
+    /**
+     * @Then there is a meta tag :arg1 with values :arg2 and :arg3
+     */
+    public function thereIsMetaTagWithValues($arg1, $arg2, $arg3)
+    {
+        $metaNode = $this->minkContext->getSession()->getPage()->find('xpath', "//meta[@name='$arg1' and @content='$arg2, $arg3']");
+        PHPUnit_Framework_Assert::assertNotNull($metaNode);
+    }
```

Comment:
Maybe this `public function thereIsMetaTagWithValues($arg1, $arg2, $arg3)` function is not required? Scenario steps that use this function, e.g. `Then There is a meta tag "robots" with values "index" and "nofollow"` can use the function above `public function thereIsAMetaTagWithValue($arg1, $arg2)` and be re-written as `Then There is a meta tag "robots" with value "index, nofollow"`.

Reply:
This step has been replaced and the behat test passed.